Sources/ProjectSpec/Project.swift

```diff
@@ -155,14 +155,7 @@ extension Project: Equatable {
 extension Project {
 
     public init(path: Path) throws {
-        var cachedSpecFiles: [Path: SpecFile] = [:]
-        let spec = try SpecFile(filePath: path, basePath: path.parent(), cachedSpecFiles: &cachedSpecFiles)
-        try self.init(spec: spec)
-    }
-
-    public init(path: Path, basePath: Path) throws {
-        var cachedSpecFiles: [Path: SpecFile] = [:]
-        let spec = try SpecFile(filePath: path, basePath: basePath, cachedSpecFiles: &cachedSpecFiles)
+        let spec = try SpecFile(path: path)
```

Comment:
This is revering a change from https://github.com/yonaskolb/XcodeGen/pull/1275. Internally the init creates the `cachedSpecFiles` so callers don't have to make it.